.github/workflows/npm-publish.yml

```diff
@@ -10,9 +10,11 @@ jobs:
       # Setup .npmrc file to publish to npm
       - uses: actions/setup-node@v4
         with:
-          node-version: '20.x'
+          node-version: '22.x'
           registry-url: 'https://registry.npmjs.org'
       - run: npm ci
+      - name: Disable AppArmor restriction
+        run: echo 0 | sudo tee /proc/sys/kernel/apparmor_restrict_unprivileged_userns
```

Comment:
I'm curious: what happens if we omit this?

Reply:
Commit name should be self-descriptive :)